src/main/java/org/dita/dost/reader/DitaValReader.java

```diff
@@ -210,6 +210,18 @@ public void readProp(final Element elem) {
 
   private Flag readFlag(Element elem) {
     final String style = getValue(elem, ATTRIBUTE_NAME_STYLE);
+    var outputClass = getValue(elem, ATTRIBUTE_NAME_ADD_OUTPUTCLASS);
+    if (outputClass == null) {
+      outputClass = getValue(elem, ATTRIBUTE_NAME_OUTPUTCLASS);
+      if (outputClass != null) {
+        logger.warn(
```

Comment:
This warning doesn't contain file location because we don't generate `@xtrf` and `@xtrc` attributes to DITAVAL elements. Still, I think it makes sense to warn about this.